executors.go

```diff
@@ -70,6 +122,29 @@ func (c *Connection) Save(model interface{}, excludeColumns ...string) error {
 // ValidateAndCreate applies validation rules on the given entry, then creates it
 // if the validation succeed, excluding the given columns.
 func (c *Connection) ValidateAndCreate(model interface{}, excludeColumns ...string) (*validate.Errors, error) {
+	v := reflect.Indirect(reflect.ValueOf(model))
+	if v.Kind() == reflect.Slice || v.Kind() == reflect.Array {
+		var err error
+		for i := 0; i < v.Len(); i++ {
+			val := v.Index(i)
+			if val.Kind() != reflect.Ptr {
+				val = val.Addr()
+			}
+			verrs, err := c.ValidateAndCreate(val.Interface(), excludeColumns...)
+			if err != nil {
+				return verrs, err
+			}
+			if verrs.HasAny() {
+				return verrs, nil
+			}
+		}
+		return validate.NewErrors(), err
+	}
+
+	if c.eager {
```

Comment:
would this statement be better at the beginning instead of after all the reflection work?